sdk/monitor/azure-monitor-query/azure/monitor/query/_helpers.py

```diff
@@ -16,14 +16,12 @@
 
```

Comment:
does _helpers_async.py need this same update?

Reply:
Yea, it does. Good catch! Completely forgot to update that as well. I am now realizing that `get_authentication_policy` and `get_metrics_authentication_policy` are exactly the same now, so will remove one in each.